SynonymCreator/src/main/scala/no/nrk/samnorsk/synonymmapper/Language.scala

```diff
@@ -0,0 +1,29 @@
+package no.nrk.samnorsk.synonymmapper
```

Comment:
I would make Language a sealed trait, this will give you compiler helt when you add or remove something from the list... If you miss an option in a match in the code the compiler will not scream at you... With a sealed trait the compiler checks if you actually match on all the languages and not just some of them.